Sources/Toolbox/Utilities/SwiftUI/ImagePicker.swift

```diff
@@ -0,0 +1,79 @@
+
+import AVFoundation
+import SwiftUI
+import UIKit
+
+@available(iOS 13.0, *)
+struct ImagePicker: UIViewControllerRepresentable {
+    class PickerCoordinator: NSObject, UINavigationControllerDelegate, UIImagePickerControllerDelegate {
+        let parent: ImagePicker
+
+        init(_ parent: ImagePicker) {
+            self.parent = parent
+        }
+
+        func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]) {
+            if let editedImage = info[.editedImage] as? UIImage {
+                parent.image = editedImage
+            }
+
+            parent.presentationMode.wrappedValue.dismiss()
+        }
+    }
+
+    @Binding var image: UIImage?
+    @Binding var sourceType: UIImagePickerController.SourceType
+    @SwiftUI .Environment(\.presentationMode) var presentationMode
+
+    let picker = UIImagePickerController()
```

Comment:
should use PHPickerViewController for iOS 14

Reply:
yes, I need to find/write a good example of it for `UIKit` and `SwiftUI` but now that ist for `iOS13` and `SwiftUI 1.0`